src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.html

```diff
@@ -32,7 +32,7 @@
             </div>
           </div>
           <div class="row mb-3">
-            <label class="col-sm-2 col-form-label">Section Name:</label>
+            <label for="sectionname" class="col-sm-2 col-form-label">Section Name:</label>
```

Comment:
Should we be consistent with previous `label`s and use `for="section-name"`?

Reply:
I notice that this is following the `id` of the input below. Should we modify the input's `id`s to use `-`?

Reply:
Good catch, will modify the IDs here.